### DISCUSSION TITLE
Enhancing the stutter function

### DISCUSSION THREAD

**Discussion by n3ptun3s** [SynthstromAudible/DelugeFirmware#231] [Ideas]
I really love the ability to stutter the audio buffer of each track.

It would be very handy and increase the performance value, if we could:
- map stutter of/off to a MIDI note
- map stutter time to MIDI cc
- have the option to quantize the stutter time
- have a blinking indicator of the stutter golden knob LED, showing the quantization 
- have an option in the community feature page for the standard quantization (eg. 1/8, 1/16, 1/32, triplets, etc)

**Comment by soymonitus** [comment]
I have plans to work on this next

**Comment by manysounds** [comment]
Implementing a stutter function similar to the new delay mode would be great.
As in: on gold knob press switch to a limited range of slice types 1-bar through 64th notes including dotted notes and triplets, possibly selectable through a menu item.

**Comment by sichtbeton** [comment]
- ability to record stutter like other fx mods in clips and in arranger globally 
- avoid clicks (like any decent beat repeat)

**Comment by worker-gray** [comment]
stutter per instrument might be fun, if not impossible

**Comment by nilscreates** [comment]
and **stutter automation** please!! would make for such an awesome addition to intuitively spice up beats and samples. Essentially a bit like Ableton's Beat Repeat (but without parametric controls)

**Comment by brawnybeats** [comment]
Thanks for working on the stutter automation, that along with quantisation is a superb addition for the like of me who like to mangle breaks as much as possible on the fly. 

What would thoughts be on adding a variable stutter length/off time (so it would essentially become more like a gating effect)?
